### PR TITLE
TC-233 Modified admin.pl to support the postgres superuser

### PR DESCRIPTION
**Pull request by dewrich**

### CODE REVIEW

File: traffic_ops/app/db/admin.pl [hunk 1]
```diff
@@ -29,23 +29,44 @@
 use YAML qw(LoadFile);
 use DBIx::Class::Schema::Loader qw/make_schema_at/;
 
+use Env;
+use Env qw(HOME);
```
Comment (dangogh) [inline traffic_ops/app/db/admin.pl]: is it worth bringing in this module for a single env var?

File: traffic_ops/app/db/admin.pl [hunk 2]
```diff
@@ -29,23 +29,44 @@
...
 	. "Purpose:  This script is used to manage database. The environments are\n"
 	. "          defined in the dbconf.yml, as well as the database names.\n\n"
-	. "arguments:   \n\n"
+	. "NOTE: \n"
+	. "Postgres Superuser: The 'postgres' superuser needs to be created to run $PROGRAM_NAME and setup databases.\n"
+	. "If the 'postgres' superuser has not been created or password has not been set then run the following commands accordingly. \n\n"
+	. "Create the 'postgres' user as a super user (if not created):\n\n"
+	. "     \$ createuser postgres --superuser --createrole --createdb --login --pwprompt\n\n"
+	. "Postgres Password: file allows for easy command line access by defaulting the user and password for the database\n"
+	. "without prompts.\n\n"
```
Comment (dangogh) [inline traffic_ops/app/db/admin.pl]: didn't understand previous 2 lines until I saw mention of .pgpass below.   That probably should be more clear

File: traffic_ops/app/db/admin.pl (hunk 1 above)
Comment (dangogh) [inline traffic_ops/app/db/admin.pl]: and why twice?

Comment (dewrich) [inline traffic_ops/app/db/admin.pl]: yea oversite, fixed it

Comment (dewrich) [inline traffic_ops/app/db/admin.pl]: Actually, I pulled that in from this Perl Doc http://perldoc.perl.org/Env.html

Comment (dewrich) [inline traffic_ops/app/db/admin.pl]: Perl also fails the compile without it

**Pull request closed without merging**